Aliucord/src/main/java/com/aliucord/coreplugins/rn/Patches.kt

```diff
@@ -150,6 +158,27 @@ fun patchDefaultAvatars() {
     )
 }
 
+fun patchUsername() {
+    Patcher.addPatch(
+        `AuthUtils$createDiscriminatorInputValidator$1`::class.java.getDeclaredMethod("getErrorMessage", TextInputLayout::class.java),
+        InsteadHook.DO_NOTHING
```

Comment:
What about those still with a legacy discrim + nitro? Or can you not even change discrim at all anymore

Reply:
Yeah you can't update your discrim anymore, it'll only get updated when your nitro runs out.

Reply:
Not true, you can still change it with Nitro

Reply:
in that case this check shouldn't be disabled if the user isn't pomelo'd yet